server/src/main/java/org/opensearch/index/translog/transfer/FileTransferTracker.java

```diff
@@ -26,11 +27,12 @@
 import java.util.stream.Collectors;
 
 /**
- * FileTransferTracker keeps track of translog files uploaded to the remote translog
+ * FileTransferTracker keeps track of generational translog files uploaded to the remote translog
  */
 public class FileTransferTracker implements FileTransferListener {
 
     private final ConcurrentHashMap<String, TransferState> fileTransferTracker;
+    private final ConcurrentHashMap<Long, TransferState> generationalFilesTransferTracker;
     private final ShardId shardId;
```

Comment:
this appears to be unused, lets remove this.

Reply:
Can we please remove this? Why do we need to keep a instance variable for this as the logger is using shardId supplied by the constructor.